src/Api/Api.csproj

```diff
@@ -29,13 +29,8 @@
   </Choose>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="5.0.9" />
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.10.0" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.3.1" />
   </ItemGroup>
-
-  <ItemGroup>
-    <DotNetCliToolReference Include="Swashbuckle.AspNetCore.Cli" Version="4.0.1" />
-  </ItemGroup>
-
```

Comment:
do we not need this for building swagger files anymore?

Reply:
No, as long as we run a `dotnet tool restore` first we are installing this as a dotnet tool instead.